### ISSUE TITLE
Anti ZnT8

### ISSUE THREAD

**Issue by MaudDLVT** [obophenotype/human-phenotype-ontology#7240]
**Preferred term label:**
anti-ZnT8 antibody positivity

**Synonyms**
Anti-Zinc transporter 8 antibody positivity, ZnT8A

**Definition (free text, please give PubMed ID)**
PMID: 32946793

**Parent term (use hpo.jax.org/app)**
HP:0030057

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
ORPHA:85138

**Your nano-attribution (ORCID)**
0000-0002-3387-1836

**Comment by pnrobinson** [comment]
added
